web/templates/confirm_dialog/confirm_archive_channel_folder.hbs

```diff
@@ -1,2 +1 @@
-<p>{{t "Channels in this folder will become uncategorized."}}</p>
-<p>{{t "This action cannot be undone."}}</p>
+{{t "Channels in this folder will become uncategorized."}} {{t "This action cannot be undone."}}
```

Comment:
I'm pretty sure these were intentionally separate paragraphs; was this design change discussed somewhere?